cmake/modules/FindBOOST.cmake

```diff
@@ -50,7 +50,9 @@ LIST(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules/)
 #
 IF(NOT Boost_FOUND AND Boost_USE_STATIC_LIBS)
   SET(Boost_USE_STATIC_LIBS FALSE)
+  LIST(REMOVE_ITEM CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules/)
   FIND_PACKAGE(Boost 1.48 COMPONENTS iostreams serialization system thread)
+  LIST(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules/)
```

Comment:
So if I understand this right, you want to use cmake's prepackaged FindBoost rather than the one in the deal.II directory? I think a comment would be useful here to explain why you do these acrobatics.